Content.Shared/Burial/GraveComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Robust.Shared.Audio;
+
+namespace Content.Shared.Burial;
+
+[RegisterComponent]
+public sealed partial class GraveComponent : Component
+{
+    /// <summary>
+    /// How long it takes to dig this grave, without modifiers
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public TimeSpan DigDelay = TimeSpan.FromSeconds(15);
+
+    /// <summary>
+    /// Modifier if digging yourself out by hand if buried alive
+    /// </summary>
+    [DataField, ViewVariables(VVAccess.ReadWrite)]
+    public float DigOutByHandModifier = 0.1f;
+
```

Comment:
Add a TODO on this for tools because adding hard-coded modifiers to the component is bad.